test/whitePaperModel/mining/borrowMiningTests.spec.ts

```diff
@@ -1188,6 +1195,238 @@ contract("borrowMiningTests", async (accounts) => {
+                                from: user1,
+                            });
+                            // 4. Claim the minted tokens
+                            // fastforward
+                            const balFIN1 = await erc20FIN.balanceOf(user2);
+                            console.log("balFIN1", balFIN1.toString());
+                            await savingAccount.fastForward(100000);
+                            // Deposit an extra tokens to create a new rate check point
+                            await savingAccount.deposit(erc20USDT.address, new BN(10), {
+                                from: user1,
+                            });
+
+                            // FIN balance before claim
+                            const claimableUSDTAmountUser2 = new BN(
+                                await savingAccount.claimForToken.call(addressUSDT, {
+                                    from: user2,
+                                })
+                            );
+                            console.log(
+                                "claimableUSDTAmountUser2",
+                                claimableUSDTAmountUser2.toString()
+                            );
+
+                            const claimableETHAmountUser2 = new BN(
+                                await savingAccount.claimForToken.call(ETH_ADDRESS, {
+                                    from: user2,
+                                })
+                            );
+                            console.log(
+                                "claimableDepositAmountUser2",
+                                claimableETHAmountUser2.toString()
+                            );
+
+                            const claimableUSDTAmountUser1 = await savingAccount.claimForToken.call(
+                                addressUSDT,
+                                {
+                                    from: user1,
+                                }
+                            );
+                            console.log(
+                                "claimableUSDTAmountUser1",
+                                claimableUSDTAmountUser1.toString()
+                            );
+
+                            let totalClaimUser2 = new BN(
+                                await savingAccount.claim.call({ from: user2 })
+                            );
+                            console.log("totalClaimUser2", totalClaimUser2.toString());
+
+                            let totalClaimUser1 = await savingAccount.claim.call({ from: user1 });
+                            console.log("totalClaimUser1", totalClaimUser1.toString());
+
+                            // expect(
+                            //     claimableUSDTAmountUser1.add(claimableETHAmountUser2)
+                            // ).to.be.bignumber.equal(totalClaimUser1);
+                            expect(
+                                claimableUSDTAmountUser2.add(claimableETHAmountUser2)
```

Comment:
both `claimableUSDTAmountUser2` and `claimableETHAmountUser2 ` should be greater than zero.

Reply:
have added a check for this